content/doc/pipeline/index.adoc

```diff
@@ -1,981 +1,437 @@
+Node::
+    In pipeline coding contexts, as opposed to Jenkins generally, a “node” is a step that does two things, typically by enlisting help from available executors on agents:
+    * Schedules the steps contained within it to run by adding them to the Jenkins build queue (so that as soon as an executor slot is free on a node, the appropriate steps run). 
+    * Creates a workspace, meaning a file directory specific to a particular job, where resource-intensive processing can occur without negatively impacting your pipeline performance. Workspaces last for the duration of the tasks assigned to them.
 
-![Pipeline Editor](images/2-workflow-edit-hr.png)
 
-## Using the Snippet Generator
-To make editing your pipelines easier, use the Snippet Generator. The Snippet Generator is dynamically populated with the latest Pipeline steps. Depending on the plugins installed in your environment, you may see more available steps.
+(In Jenkins generally, a “node” means any computer that is part of your Jenkins installation, whether that computer is used as a master or as an agent).
 
-![Snippet Generator](images/3-snippet-generator-hr.png)
+Stage::
+    A “stage” is a step that calls supported APIs. Pipeline syntax is comprised of stages. Each stage can have one or more build steps within it. 
 
-## Loading External Pipeline Scripts
+Familiarity with Jenkins  terms such as “master,” “agent,” and “executor” also helps with understanding how pipelines work. These terms are not specific to pipelines:
 
-Because pipelines are text assets, they are ideal to store in a source control system. Pipelines can be edited in your external IDE then loaded into Jenkins using the Pipeline Script from SCM option.
+* master - A “master” is the basic installation of Jenkins on a computer; it handles tasks for your build system. Pipeline scripts are parsed on masters, and steps wrapped in node blocks are performed on available executors.
+* agent - An “agent” (formerly "slave")  is a computer set up to offload particular projects from the master. Your configuration determines the number and scope of operations that an agent can perform. Operations are performed by executors.
+* executor - An “executor” is a computational resource for compiling code. It can run on master or agent machines, either by itself or in parallel with other executors. Jenkins assigns a _java.lang.Thread_ to each executor.
 
-# Building your Pipeline
+== Preparing Jenkins to Run Pipelines
 
-Now that you've created a pipeline, let's continue to build on it. For a complex flow, you should leverage Jenkins' job scheduling queue:
+To run pipelines, you need to have a Jenkins instance that is set up with the appropriate plugins. This requires:
 
-```groovy
-node{ sh 'uname' }
-```
+* Jenkins 1.580.1 or later (Jenkins 2.0 is recommended)
+* The core Pipeline plugin
```

Comment:
> The core Pipeline plugin

There is no such "core Pipeline plugin" in the update center, I understand conceptually why this is here, but from a new user standpoint "core" means nothing. It should just be referred to as:

> The Pipeline plugin